custom/icds_reports/utils/aggregation_helpers/ccs_record_monthly.py

```diff
@@ -157,25 +234,29 @@ def aggregation_query(self):
             ('closed', 'case_list.closed'),
             ('anc_abnormalities', 'agg_bp.anc_abnormalities'),
             ('home_visit_date', 'agg_bp.latest_time_end_processed'),
-            ('date_death', 'person.date_death'),
-            ('person_case_id', 'ucr.person_case_id')
+            ('date_death', 'person_cases.date_death'),
+            ('person_case_id', 'case_list.person_case_id')
         )
         return """
         INSERT INTO "{tablename}" (
             {columns}
         ) (SELECT
             {calculations}
-            FROM "{ucr_ccs_record_monthly_table}" ucr
-            LEFT OUTER JOIN "{agg_thr_table}" agg_thr ON ucr.doc_id = agg_thr.case_id AND ucr.month = agg_thr.month and ucr.valid_in_month = 1
-            LEFT OUTER JOIN "{agg_bp_table}" agg_bp ON ucr.doc_id = agg_bp.case_id AND ucr.month = agg_bp.month and ucr.valid_in_month = 1
-            LEFT OUTER JOIN "{agg_pnc_table}" agg_pnc ON ucr.doc_id = agg_pnc.case_id AND ucr.month = agg_pnc.month and ucr.valid_in_month = 1
-            LEFT OUTER JOIN "{agg_cf_table}" agg_cf ON ucr.doc_id = agg_cf.case_id AND ucr.month = agg_cf.month and ucr.valid_in_month = 1
-            LEFT OUTER JOIN "{agg_delivery_table}" agg_delivery ON ucr.doc_id = agg_delivery.case_id AND ucr.month = agg_delivery.month and ucr.valid_in_month = 1
-            LEFT OUTER JOIN "{ccs_record_case_ucr}" case_list ON ucr.doc_id = case_list.doc_id
-            LEFT OUTER JOIN "{pregnant_tasks_case_ucr}" ut ON ucr.doc_id = ut.ccs_record_case_id
-            LEFT OUTER JOIN "{person_cases_ucr}" person ON person.doc_id = ucr.person_case_id
-            WHERE ucr.month = %(start_date)s
-            ORDER BY ucr.awc_id, ucr.case_id
+            FROM "{ccs_record_case_ucr}" case_list
```

Comment:
I think you want to add filter to only include people opened before the end of this month. Otherwise there will be rows from future cases

Reply:
Sure,I can do that but that will lead to deletion of all the rows from output data which have open_data > end of iteration month. Does that sound right

Reply:
yes there does exists data from future in `ccs_record_monthly` table. Running this query 
`select pregnant,valid_in_month,opened_on from "ccs_record_monthly_2018-11-01" where add>'2018-11-30' limit 1`
Ideally should return no rows. But it has the records opened in month `Dec` even though this is the table for `Nov`.

So, If we think this should not be the case from this change onward. I can correct it by adding filters. But that would need deletion of future rows from output test data
@calellowitz